packages/circuit-ui/components/NotificationFullscreen/NotificationFullscreen.tsx

```diff
@@ -63,6 +63,26 @@ const imageStyles = css`
   object-fit: contain;
 `;
 
+const svgStyles = css`
+  height: 100%;
+  width: 100%;
+`;
+
+function NotificationImage({
+  image,
+}: Pick<NotificationFullscreenProps, 'image'>) {
+  if (isFunction(image)) {
+    const Svg = image;
+    return (
+      <div css={imageStyles}>
+        <Svg css={svgStyles} />
```

Comment:
One downside of this approach, compared to the `ImageProps` one, is that there's no required alternative text for the asset. On the other hand, for notification SVGs (at least in the SumUp context), it's likely that the asset will only be illustrative and should have `aria-hidden="true"`.

Were you thinking of leaving this concern to developers?

Reply:
Good point! I actually didn't think of that 🙈 

Would you prefer an optional alt text prop or simply adding `aria-hidden="true"`/`role="presentation"` by default? (I haven't seen any use cases where an alt text would have been necessary)

Reply:
> Would you prefer an optional alt text prop or simply adding aria-hidden="true"/role="presentation" by default?

My thoughts 👇 

### Decorative media (by default)

Adding `aria-hidden` or a presentation role to the SVG is probably the "cleanest" and would work in most (all?) cases today, but I'm not sure we can/should assume that SVGs will always be decorative.

If they're only decorative _by default_, this might be a good option. What API would you suggest to override the default and add alternative text or a label? The option I'm thinking of is quite verbose.

Also, if we go down that route, I'd also make the `img` decorative (for consistency).

### Alt text

I like this option as well but it might also make the API a bit messy—at least with the option i'm thinking of:

```ts
type Props = {
  image?: {
    src?: string,
    children?: FC<SVGProps<SVGSVGElement>>,
    alt: string, // must always be provided, even if it's `""` for a decorative image
  },
  /* etc */
}
```

My thinking:
- This is nice because alternative text is always required, no matter if the actual asset is an `img` or an `svg`
- ...on the other hand `alt` isn't appropriate on an `svg`, so internally we should check which of the two elements is rendered and either use `alt` or `aria-label`
- BUT! I really don't like APIs like this where you could pass either an src or children—it makes it confusing. But it would be equally confusing to have something like `src: string | FC<SVGProps<SVGSVGElement>>`
- ...and finally I don't like how this is moving away from the `ImageProps`

Did you have other APIs in mind? Right now I'm not convinced by any solution 😅

Reply:
Alright, so want to have the option to provide alternative text, but by default, the image/illustration should be hidden from screen reader users.

> `alt` isn't appropriate on an `svg`, so internally we should check which of the two elements is rendered and either use `alt` or `aria-label`

Yes. I would stick to `alt` as the prop name as the `aria-label` is also "alternative text" and developers are already familiar with its purpose.

I would call the SVG something other than `children` since that would imply a `ReactNode`, but we're expecting a `ReactElement`. So `element` perhaps? Or `svg`? 🤔

>  I don't like how this is moving away from the `ImageProps`

Eh, I feel you, but I think there's no better approach to this.

So, we end up with: 

```ts
type Props = {
  image?: {
    src: string,
    alt?: string, // defaults to `""`
  } | {
	children: FC<SVGProps<SVGSVGElement>>,
    alt?: string, // defaults to `""`
  },
  /* etc */
}
```

Reply:
I'm thinking of going for this:

```ts
type Props = {
  image: ImageProps | { svg: FC<SVGProps<SVGSVGElement>>, alt: string },
  /* etc */
}
```

Reasoning here is that although the image will often be decorative, I'd still like to consistently require the alt prop when we render graphics across Circuit UI, just to make 100% sure that developers think about it. It'll often be `alt=""` and that's okay 🙂 

Also the naming "svg" is a good choice here IMO: we need the element to be an SVG so I think it doesn't make sense to make the naming broader (e.g. "element").

Let me know if you have concerns!